config/settings.yml

```diff
@@ -912,6 +912,13 @@
   :allow_api_service_ordering: true
 :prototype:
   :messaging_type: miq_queue
+:rate_limiting:
+  :default:
```

Comment:
Can we change this to `requests`? `default` makes it sound like it is a base value for `login` (and other types), but that's now how it's used.

Reply:
I was just trying to map to the name in `.throttle` to the name of our setting, but I have no real attachment to what I picked.

Reply:
Err... nevermind, I thought this was something else.  Sure, will change.

Reply:
Fixed.